Frameworks/UIKit/UIViewController.mm

```diff
@@ -1207,43 +1196,40 @@ - (void)presentViewController:(UIViewController*)controller animated:(BOOL)anima
 
     [[UIApplication sharedApplication] beginIgnoringInteractionEvents];
 
-    if ([controller _hidesParent]) {
-        [self _notifyViewWillDisappear:animated];
-    }
-
-    priv->_presentedViewController = controller;
     [controller view];
```

Comment:
can you add a comment here so we know not to reorder it in the future?

---
In reply to: [106151392](https://github.com/Microsoft/WinObjC/pull/2195#discussion_r106151392) [](ancestors = 106151392)